sdk/loadtestservice/Azure.Developer.LoadTesting/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please write the changelog. please refer to changelog guideline: https://azure.github.io/azure-sdk/policies_releases.html#change-logs

Write the changes and fit to right blocks(Features Added, Breaking Changes and so on). If the change block does not contain content, remove the block.

Reply:
corrected changelog!